h2o-persist-s3/src/main/java/water/persist/PersistS3.java

```diff
@@ -439,6 +439,7 @@ public Key uriToKey(URI uri) throws IOException {
 
     public boolean containsKey(String k) { return Arrays.binarySearch(_cache,k) >= 0;}
     protected String [] update(){
+      Log.debug("Renewing S3 bucket cache.");
```

Comment:
maybe next time use Log4j

Reply:
https://github.com/h2oai/h2o-3/pull/4682 Fixing it now. Thanks.